app/jobs/state_file/after_transition_tasks_for_rejected_return_job.rb

```diff
@@ -6,7 +6,11 @@ def perform(submission, transition)
       Efile::SubmissionErrorParser.persist_errors(transition)
 
       if transition.efile_errors.any?
-        submission.transition_to!(:cancelled) if transition.efile_errors.any?(&:auto_cancel)
+        if transition.efile_errors.any?(&:auto_cancel)
+          submission.transition_to!(:notified_of_rejection)
+          submission.transition_to!(:cancelled)
+        end
```

Comment:
remind me whether you ran this by Tiffany? i think it makes sense to me but the notified of rejection state is still slightly confusing to me so we should make sure this is sanctioned by product

Reply:
Tiffany approved of this! Just demo'd on tuple.